mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenProjectConfigurator.java

```diff
@@ -88,7 +87,8 @@ void configureTestRunner(Model model) {
 
         return testRunnerPluginConfigurations.stream()
             .filter(
-                testRunnerPlugin -> !(testRunnerPlugin.getArtifactId().equals("maven-failsafe-plugin") && isSkipITs()))
+                testRunnerPlugin -> !(testRunnerPlugin.getArtifactId().equals("maven-failsafe-plugin")
+                    && new MavenPropertyResolver(model).isSkipITs()))
```

Comment:
Instead of using the string `"maven-failsafe-plugin"` you could use `ApplicablePlugins.FAILSAFE.hasSameArtifactId(testRunnerPlugin.getArtifactId())`

In additon, it took me some time to understand what the if 
`!(testRunnerPlugin.getArtifactId().equals("maven-failsafe-plugin")
&& new MavenPropertyResolver(model).isSkipITs())`
represents. move either the if or the whole filter stream to a seperated method. eg:
`removeFailsafePluginIfSkipped(List<Plugin> testRunnerPluginConfigurations, Model model)`

Reply:
Btw, in the case when the failsafe plugin is removed, then [this if](https://github.com/arquillian/smart-testing/pull/211/files#diff-5a46d71d79f00752a476439e3186179eR38) will return `false`, but the message about disabling ST for the module is not printed.

Reply:
And yeah, shouldn't we do the same thing also for the surefire plugin? otherwise it will configure the ST for surefire no matter what is set there - in the case when both surefire and failsafe plugins are specified and both of them are skipped.